build.sh

```diff
@@ -57,4 +57,6 @@ fi
 
 [ "${PORTS_MICROPYTHON}" = "y" ] && ./phoenix-rtos-ports/micropython/build.sh
 
+[ "${PORTS_WPA_SUPPLICANT}" = "y" ] && ./phoenix-rtos-ports/wpa_supplicant/build.sh
```

Comment:
For me compilation fails with:

```
  CC  ../src/crypto/tls_openssl.c
../src/l2_packet/l2_packet_linux.c:11:10: fatal error: netpacket/packet.h: No such file or directory
   11 | #include <netpacket/packet.h>
```

do I need some other changes for this to work?

Reply:
It doesn't build without additional changes in libphoenix, phoenix-rtos-kernel and phoenix-rtos-lwip (not available yet).

Reply:
ok, changing to draft PR to avoid accidental merge by someone else :)

Reply:
is this ready for review/merge now? If so, please mark it as non-draft PR